Scripts/test-exit.py

```diff
@@ -1,9 +1,33 @@
+## This small script takes as first argument a test indicator
+## and the rest of the arguments are a command to run, with its
+## command-line arguments.
+## The script runs the command and then if the first argument is
+## (1) -ok -- returns success, ignoring the exit code of the command
```

Comment:
Perhaps `-any` would be more accurate? If `-ok` is more idiomatic for scripts ignore this. :)